src/containers/CategoriesContainer.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Created by Katarzyna_Bak on 19.05.2017.
+ */
+
+import { connect } from 'react-redux';
+import Categories from './../Categories';
+import { addCategory, addSubcategory, deleteCategory, editCategory, deleteTask } from './../actions';
+
+const mapStateToProps = (state) => {
+  return {
+      categories: state.categories
+  }
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+      onAddCategory: (text) => {
+          dispatch(addCategory(text));
+      },
+      onAddSubcategory: () => {
+          dispatch(addSubcategory());
+      },
+      onDeleteCategory: (id) => {
+          dispatch(deleteCategory(id));
+          dispatch(deleteTask(id));
+      },
+      onEditCategory: () => {
+          dispatch(editCategory());
+      }
+  }
+};
```

Comment:
`mapDispatchToProps` accepts object too (check [Redux docs](https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments) for details):

```js
const mapDispatchToProps = {
  onAddCategory: addCategory,
  onAddSubcategory: addSubcategory,
  onDeleteCategory: deleteCategory, // TODO update deleteCategory to remove it's tasks as well
  onEditCategory: editCategory,
};
```